spec/passaporte_web/identity_spec.rb

```diff
@@ -332,9 +332,9 @@
             send_myfreecomm_news: true
           }
           identity = described_class.new(attributes)
-          mock_response = mock('response', code: 201, body: MultiJson.encode(attributes))
+          mock_response = double('response', code: 201, body: MultiJson.encode(attributes))
```

Comment:
Line is too long. [91/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.